pkgs/development/haskell-modules/configuration-ghc-7.8.x.nix

```diff
@@ -75,6 +75,9 @@ self: super: {
   ghc-exactprint = dontDistribute super.ghc-exactprint;
   ghc-typelits-natnormalise = dontDistribute super.ghc-typelits-natnormalise;
 
+  # Needs directory >= 1.2.2.0.
+  idris = dontDistribute super.idris;
```

Comment:
The main point was to stop my Hydra bulding it (it builds 7.8.4 packages for me); though I don't need 7.8.4 much now anyway, so I should probably just disable builds altogether. Please pull https://github.com/abbradar/nixpkgs/commit/1e5d394439290cb124af498972cafd2ad5cce23f